cabal-testsuite/README.md

```diff
@@ -4,11 +4,12 @@ frameworks.
 How to run
 ----------
 
-1. Build `cabal-tests` (`cabal build cabal-tests`)
+1. Build `cabal-testsuite` (`cabal build cabal-testsuite`)
```

Comment:
Mhhh I wonder if this maybe could be slightly more future-proof/correct as `cabal build cabal-testsuite:cabal-tests`)?

Reply:
Fixed: f972cf89339d2265676e32c4628975c24b63f726